src/fiskaltrust.Launcher/ProcessHost/ProcessHostMonarch.cs

```diff
@@ -88,6 +94,23 @@ public Task Start(CancellationToken cancellationToken)
             {
                 _logger.LogInformation("Host {package} {id} has shutdown.", _packageConfiguration.Package, _packageConfiguration.Id);
 
+                if (_process.ExitCode != 0)
+                {
+                    foreach (var log in _plebianLogBuffer)
+                    {
+                        if (log.Contains("Error"))
+                        {
+                            _logger.LogError(log);
+                        }
+                        else
+                        {
+                            _logger.LogInformation(log);
+                        }
+                    }
```

Comment:
I'm thinking that maybe its better to print that as one large error message instead. 🤔

something like this `_logger.LogError("Error when starting {package} {id}.\n{error}", _packageConfiguration.Package, _packageConfiguration.Id, _plebianLogBuffer.Aggregate())`


Because for one this can only happen when the plebian crashed so everything part of that error.
And we keep the information which package encountered the error inside one log message.

Ignore that comment if you disagree with my take ^^

Reply:
Yes, totally agree with you. I am doing it like this. Thanks Paul